src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CompanyName.MyMeetings.BuildingBlocks.Domain;
+
+namespace CompanyName.MyMeetings.Modules.Meetings.Domain.MeetingComments.Rules
+{
+    public class CommentTextMustBeProvidedRule : IBusinessRule
+    {
+        private readonly string _comment;
+
+        public CommentTextMustBeProvidedRule(string comment)
+        {
+            _comment = comment;
+        }
+
+        public bool IsBroken() => _comment == null;
```

Comment:
Maybe `string.IsNullOrEmpty(_comment)` to check against empty string too?

Reply:
i agree, thanks